utils-cdi/src/test/java/be/atbash/util/BeanManagerFake.java

```diff
@@ -71,6 +86,10 @@ public void registerBean(String beanName, Object instance) {
         registeredBeans.put(beanName, instance);
     }
 
+    public void registerBean(AnnotationLiteral beanName, Object instance) {
+        registeredLiterals.put(beanName, new ArrayList<Object>(Arrays.asList(instance)));
```

Comment:
Collections.singletonList(instance) is preferred over Arrays.asList(instance).

Is the new ArrayList() required? (does the List needs to be modifiable)

Reply:
In fact, there's no need for "new ArrayList()"